Userland/Libraries/LibGfx/ImageFormats/JBIG2Loader.cpp

```diff
@@ -468,7 +468,7 @@ enum class CombinationOperator {
 };
 
 static void composite_bitbuffer(BitBuffer& out, BitBuffer const& bitmap, Gfx::IntPoint position, CombinationOperator operator_)
- {
+{
```

Comment:
This formatting fix should be squashed into the previous commit

Reply:
Whoops, sorry!